### DISCUSSION TITLE
Creating GitHub repositories for plugins

### DISCUSSION THREAD

**Discussion by xilopaint** [swiftbar/SwiftBar#291] [Q&A] [answered]
I plan to move my plugin projects to GitHub, but I still wonder what would be the best approach for the local repositories. Would creating the repositories inside the SwiftBar Plugin Folder be a possibility or should I do it somewhere else and symlink the plugins to the Plugin Folder?

**Comment by melonamin** [comment, marked as answer]
> Would creating the repositories inside the SwiftBar Plugin Folder be a possibility

I didn’t try this, but in theory it should work just fine. All git stuff is in .git folder, which is hidden, which means SwiftBar is going to ignore it.

Personally I would keep repositories separately, but this is just a preference

**Reply by xilopaint** [reply]
> Personally I would keep repositories separately, but this is just a preference

In this case you symlink the plugin file to the Plugin Folder?

**Reply by melonamin** [reply]
Yep
